bolt/test/AArch64/dwarf4-dwp-aarch64.s

```diff
@@ -0,0 +1,453 @@
+# RUN: rm -rf %t && mkdir -p %t && cd %t
```

Comment:
Please add description of what this test does.
I think current convention is to use ## for Comments.